install_config/redeploying_certificates.adoc

```diff
@@ -623,17 +623,67 @@ EOF
 oc create -f -
 ----
 +
-Then, run the following to remove the environment variables:
+Then, run the following command to remove the environment variables:
 +
 ----
 $ oc env dc/router OPENSHIFT_CA_DATA- OPENSHIFT_CERT_DATA- OPENSHIFT_KEY_DATA- OPENSHIFT_MASTER-
 ----
 
-. Delete the `router-certs` secret:
+. Obtain a certificate.
+* If you use an external Certificate Authority (CA) to sign your certificates,
+create a new certificate and provide it to {product-title} by following your
+internal processes.
+* If you use the internal {product-title} CA to sign certificates, run the
+following commands:
++
+[IMPORTANT]
+====
+The following commands generate a certificate that is internally signed. It will
+be trusted by only clients that trust the {product-title} CA.
+====
++
+----
+$ cd /root
+$ mkdir cert ; cd cert
+$ oc adm ca create-server-cert \
+    --signer-cert=/etc/origin/master/ca.crt \
+    --signer-key=/etc/origin/master/ca.key \
+    --signer-serial=/etc/origin/master/ca.serial.txt \
+    --hostnames='*.hostnames.for.the.certificate' \
+    --cert=router.crt \
+    --key=router.key \
+
+----
++
+These commands generates the following files:
```

Comment:
s/generates/generate

Although, you could just as easily say "This generates the following files:"

Reply:
+1 on the first

I was taught to always qualify "this" and "these" to avoid ambiguity.